packages/react-native-reanimated/src/layoutReanimation/animationsManager.ts

```diff
@@ -15,14 +15,12 @@ const TAG_OFFSET = 1e9;
 
 function startObservingProgress(
   tag: number,
-  sharedValue: SharedValue<Record<string, unknown>>,
-  animationType: LayoutAnimationType
+  sharedValue: SharedValue<Record<string, unknown>>
 ): void {
   'worklet';
-  const isSharedTransition =
-    animationType === LayoutAnimationType.SHARED_ELEMENT_TRANSITION;
   sharedValue.addListener(tag + TAG_OFFSET, () => {
-    global._notifyAboutProgress(tag, sharedValue.value, isSharedTransition);
+    // @ts-ignore: Temporary solution to avoid changes in C++ common code.
```

Comment:
Perhaps it's better to remove the third parameter in C++ now instead of using `@ts-ignore`?

Reply:
It is removed in following PR - https://github.com/software-mansion/react-native-reanimated/pull/7119